server/src/test/java/org/opensearch/cluster/decommission/DecommissionServiceTests.java

```diff
@@ -195,6 +199,43 @@ public void onFailure(Exception e) {
         assertTrue(countDownLatch.await(30, TimeUnit.SECONDS));
     }
 
+    public void testExternalDecommissionRetryNotAllowed() throws InterruptedException {
```

Comment:
need an integ test to show retries are working

Reply:
Sure, working on it.

Reply:
Added